templates/quickstart-vpc-for-atlassian-services.yaml

```diff
@@ -8,13 +8,11 @@ Metadata:
           default: ' VPC Network Configuration'
         Parameters:
           - AccessCIDR
-          - AssociatePublicIpAddress
           - AvailabilityZones
           - PrivateSubnet1CIDR
           - PrivateSubnet2CIDR
           - PublicSubnet1CIDR
           - PublicSubnet2CIDR
-          - SSLCertificateName
```

Comment:
Why did we need to remove these parameters?

Reply:
They are not used in the template at all. They are some leftovers.
They are also not mentioned in the guide as well - https://aws-quickstart.s3.amazonaws.com/quickstart-atlassian-services/doc/atlassian-standard-infrastructure-on-the-aws-cloud.pdf